### PR TITLE
Don't recommend overriding _isAllowed

### PR DESCRIPTION
**Pull request by mpchadwick**
## Purpose of this pull request

Recommend the better way for binding a controller action to an admin resource

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.html

## Links to Magento source code

Declaring `ADMIN_RESOURCE` is the better way and is how it's done in core.

https://github.com/magento/magento2/blob/2.4.2/app/code/Magento/Tax/Controller/Adminhtml/Tax.php#L23
https://github.com/magento/magento2/blob/2.4.2/app/code/Magento/Rss/Controller/Adminhtml/Feed.php#L17
https://github.com/magento/magento2/blob/2.4.2/app/code/Magento/Cms/Controller/Adminhtml/Block.php#L15

There's no need to override `_isAllowed` as it will check that const

https://github.com/magento/magento2/blob/2.4.2/app/code/Magento/Backend/App/AbstractAction.php#L213

### CODE REVIEW

File: src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md [hunk 1]
```diff
@@ -96,26 +96,19 @@ The menu displays as follows:
 
 ### Restrict admin controllers
 
-We can restrict the access to admin controllers by overriding the `_isAllowed` method of the `\Magento\Backend\App\Action` class.
+We can restrict the access to admin controllers by setting the `ADMIN_RESOURCE` const.
```
Comment (BarnyShergold) [inline src/guides/v2.3/ext-best-practices/tutorials/create-access-control-list-rule.md:99]: Please don't use contractions in text.
```suggestion
We can restrict the access to admin controllers by setting the `ADMIN_RESOURCE` constant.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Nice update - just one small change

**Comment by mpchadwick** [comment]
@BarnyShergold Change applied

**Review by BarnyShergold — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dobooth the pull request successfully imported.

**Pull request merged**